action.yml

```diff
@@ -17,6 +17,15 @@ inputs:
     description: If the release should be a draft
     required: true
     default: false
+  tag-as-name:
```

Comment:
:thinking: I'm going to brainstorm and see if there's a more descriptive name. Can't think of one now, but I wonder if there's an input name that can capture both
1. the tag's name becomes the title
2. the *entire* tag's contents becomes the body

But, at least, I think that `title` is more accurate than `name`. If you use `gh release create`, for example, you're prompted for the release's *title.*

Reply:
You are right. I used the term `title` everywhere at first, but was discouraged, when I saw that the [Github REST API](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release) defines the field for release creation as `name`. So I changed everything to `name`. I guess it is an inconsistency on Github's part. If you like `title` more as well, I am all for it. But then you may want to use it throughout the existing code as well instead of `name`.

As for the option name, I doubt it makes sense to try and convey _both_ in it. I think `tag-as-title` indicates the first point, which is what I initially wanted to happen. Alternatively, we could simply call that option `title` and allow it to be passed either `tag` or `subject` as arguments for now. Basically an enum. That would allow adding more choices later on. Maybe something like `both` or `tag-and-subject` to result in something like `v1.0.1 (This is the subject line)` or something to that effect. But that is just me brainstorming as well.

Reply:
Or how about this:

Introduce the `title` parameter that accepts _any_ string. But we allow special placeholders for the tag name or the subject line in it. The user can specify something like `'$tag ($subject)'` and would get `v1.0.1 (This is the subject line)` as the title. That gives maximum flexibility.

And in addition we have the option `text` with the enum `empty`, `subject`, `body`, `contents` or `contents-signed` (or something similar). Maybe overkill with that last option...

Reply:
:thinking: After thinking about it, `tag-as-title` is probably best.

> Introduce the `title` parameter that accepts _any_ string. But we allow special placeholders for the tag name or the subject line in it. The user can specify something like `'$tag ($subject)'` and would get `v1.0.1 (This is the subject line)` as the title. That gives maximum flexibility.

That's a really fun idea, but, for now, I think let's just stick with `tag-as-title: true|false`. That level of complexity is kind of what this action is trying to abstract away -- there isn't much different between `$subject` and `(contents:subject)` IMO. A sort of templating input probably belongs in a different action IMO.